src/components/Home/HomeHeaderBackground.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Box, Container } from '@mui/material';
+
+import SearchForm from './SearchForm';
+import HomeBackground from '../../assets/landscapeImage.jpg';
+
+const HomeHeaderBackground = () => {
```

Comment:
i think `Banner` is the popular term for this component. Similar to header and footer, will this be re used throughout the website?